config.js

```diff
@@ -129,6 +129,11 @@ var conf = convict({
       format: String,
       default: 'https://www.mozilla.org/firefox/ios/'
     },
+    signInUrl: {
```

Comment:
we will need to add this to https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L367 as well